src/main/java/de/zalando/ep/zalenium/registry/ZaleniumRegistry.java

```diff
@@ -251,8 +264,24 @@ public void addNewSessionRequest(RequestHandler handler) {
         try {
             lock.lock();
             Map<String, Object> requestedCapabilities = handler.getRequest().getDesiredCapabilities();
-            proxies.verifyAbilityToHandleDesiredCapabilities(requestedCapabilities);
+
+            //Prefix custom capabilities and truncate them when they are longer than 250 characters
+            for(String capability :  Arrays.asList("name" , "zal:name", "build", "zal:build")) {
```

Comment:
The prefixing should be done at the client, for example C# does not allow you to send a test request if custom capabilities are not prefixed.

Reply:
We didn't prefix nothing in the client but we found that Zalenium  somewhere added the prefix zal: to some of the capabilities and then it started to process the same request several times. We try to find the specific case.

Reply:
Yes, we add the prefix as a way to help some bindings, but in the end, this should be done in the client.